README.md

```diff
@@ -81,14 +81,12 @@ Notice that `.cargo/config.toml` has an entry for replacing the linker when cros
 linker = "arm-linux-gnueabihf-gcc"
 ```
 
-This means `arm-linux gnueabihf-gcc` must be available on the system and will be used as the linker. Once you have it installed, you can use:
+This means `arm-linux gnueabihf-gcc` must be available on the system and will be used as the linker. Once you have it installed, you can produce an ARMv7 binary using:
```

Comment:
there's a missing '-' in `arm-linux-gnueabihf-gcc`.

Reply:
Nice catch, I missed that the first time through as well. I will fix this 👍